server/players.py

```diff
@@ -76,6 +79,39 @@ def __init__(
         self._game = lambda: None
         self._game_connection = lambda: None
 
+    @property
+    def global_rating(self):
+        return self._global_rating
+
+    @global_rating.setter
+    def global_rating(self, value: Rating):
+        if isinstance(value, Rating):
+            self._global_rating = (value.mu, value.sigma)
+        else:
+            self._global_rating = value
+
+    @property
+    def ladder_rating(self):
```

Comment:
missing return typing, if it would be just variable, not property, code would be much easier to read, and for outside world it behaves the same

Reply:
As above, didn't bother changing copied code. It deserves to be a named tuple anyway, we can change it then.